tests/performance/api.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { PerformanceObserver, performance } from 'perf_hooks';
+
+import { explore } from '../../src/api';
+import { setTestFolder } from '../test-helpers';
+
+// Set to value actual for CI (not your local environment)
+const BIG_FILE_EXPLORE_WITH_HTML = 5500;
```

Comment:
How long did this take before the performance fix? Since there's going to be some variation, we should make this large enough to avoid spurious failures.

Also, suggest adding `_MS` as a suffix to indicate the units on this.

Reply:
It took ~5100ms

Reply:
Hmm, so this test would have passed before the fix? Perhaps lower the value then to make it a true regression test.

Reply:
Sorry, I thought you asking about the biggest time on CI after fix.
Before the fix time differed in a degree. On my machine:

_ | _
----------|-----------
Before: | 235362ms
After: | 1391ms

Reply:
I increase the threshold to 6000ms so that we have ~1000ms buffer from the worst result on CI